### PR TITLE
[VigiCrues] Adding localization and a bit of code cleansing.

### PR DESCRIPTION
**Pull request by clinique**
Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
+# binding
+binding.vigicrues.name = Extension VigiCrues
+binding.vigicrues.description = Informations de niveau et de seuil d'alerte des riv�res de France.
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties:4]: typo: rivières

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Station thing config
+thing-type.config.vigicrues.station.id.label = Identifiant
+thing-type.config.vigicrues.station.id.description = Identifiant unique de la station.
+
+thing-type.config.vigicrues.station.refresh.label = P�riode de Rafra�chissement
+thing-type.config.vigicrues.station.refresh.description = Fr�quence de rafraichissement en minutes.
+
+# thing types
+thing-type.vigicrues.station.label = Station
+thing-type.vigicrues.station.description = Fourni les informations de niveau et de flux pour cette station.
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties:15]: Fournit

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+channel-type.vigicrues.height.label = Hauteur
+channel-type.vigicrues.height.description = Hauteur d'eau dans le cours d'eau.
+
+channel-type.vigicrues.relative-height.label = Hauteur relative
+channel-type.vigicrues.relative-height.description = Hauteur relative par rapport aux crues historiques.
+
+channel-type.vigicrues.flow.label = D�bit
+channel-type.vigicrues.flow.description = D�bit du cours d'eau.
+
+channel-type.vigicrues.relative-flow.label = D�bit relative
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/i18n/vigicrues_fr.properties:27]: relatif

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -59,45 +59,72 @@
...
+			<tag>Timestamp</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="height">
 		<item-type>Number:Length</item-type>
 		<label>Height</label>
 		<description>Water level in the river</description>
+		<category>flowpipe</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:73]: I don't think this is an official category (with an icon in the classic iconset).

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml [hunk 5]
```diff
@@ -59,45 +59,72 @@
...
+			<tag>Level</tag>
+		</tags>
 		<state readOnly="true" pattern="%.2f %unit%"/>
 	</channel-type>
 
 	<channel-type id="alert-level">
 		<item-type>Number</item-type>
-		<label>Alerte</label>
+		<label>Alert</label>
+		<category>error</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:94]: Not an official category I believe.

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml [hunk 6]
```diff
@@ -59,45 +59,72 @@
...
+				<option value="3">Red</option>
 			</options>
 		</state>
 	</channel-type>
 
 	<channel-type id="comment">
 		<item-type>String</item-type>
-		<label>Commentaire</label>
+		<label>Comment</label>
+		<category>text</category>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:112]: Is the category really useful ?

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml [hunk 7]
```diff
@@ -59,45 +59,72 @@
...
+		<tags>
+			<tag>Status</tag>
+			<tag>Water</tag>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:115]: I am not sure that Water is appropriate for a comment ?

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml [hunk 8]
```diff
@@ -59,45 +59,72 @@
...
 
 	<channel-type id="alert-icon">
 		<item-type>Image</item-type>
-		<label>Pictogramme</label>
-		<description>Pictogramme associé au niveau d'alerte.</description>
+		<label>Pictogram</label>
+		<description>Official pictogram associated to alert level.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Water</tag>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:126]: I am not sure that Water is appropriate for an image ?

**Review by lolodomo — CHANGES_REQUESTED** [review]
Few comments

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (clinique) [reply bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:73]: Yes, it is and shows a nice blue arrow

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml (hunk 5 above)
Comment (clinique) [reply bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:94]: Yes, it is

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml (hunk 6 above)
Comment (clinique) [reply bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:112]: I shows a nice icon :)

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:73]: Verified ;)

File: bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml (hunk 5 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml:94]: Verified ;)

**Review by lolodomo — APPROVED** [review]
LGTM now

**Review by fwolter — APPROVED** [review]
LGTM

**Pull request merged**